src/js/actions/export.js

```diff
@@ -969,7 +969,7 @@ define(function (require, exports) {
     exports.exportLayerAssets = exportLayerAssets;
     exports.exportDocumentAssets = exportDocumentAssets;
     exports.setUseArtboardPrefix = setUseArtboardPrefix;
-    exports.afterStartup = afterStartup;
+    // exports.afterStartup = afterStartup;
```

Comment:
I'll take it out before i think the PR is merge-able, but i prefer to leave it in while the PR is review only since my local build doesn't have generator .